test/services/katello/pulp/repository/ostree_test.rb

```diff
@@ -22,7 +22,13 @@ def delete_repo(repo)
 
         def test_mirrored_importer
           service = Katello::Pulp::Repository::Ostree.new(@repo, @mirror)
-
+          Cert::Certs.stubs(:ueber_cert).returns({})
+          Katello::Pulp::Repository::Ostree.any_instance.stubs(:mirror_importer_connection_options).returns(
+                                                                                                         ssl_client_cert: nil,
+                                                                                                           ssl_client_key: nil,
+                                                                                                           ssl_ca_cert: nil,
+                                                                                                           ssl_validation: false
+          )
           assert_include service.generate_mirror_importer.feed, URI(SmartProxy.pulp_master.pulp_url).host
```

Comment:
This was needed cause with the existing VCR the tests seem to pass. But re-recording the cassette fails looking for the cert during the test. Stubbing it inside the method should make the re-recording more stable.